src/components/Tabs/Tabs.scss

```diff
@@ -327,24 +326,88 @@ $focus-state-box-shadow-color: rgba(0, 0, 0, 0.8);
 
   .Tab {
     padding: spacing(tight) spacing(extra-tight);
-
-    &:active .Title::before {
-      background: var(--p-action-primary-pressed);
+    &:hover .Title {
+      @media (-ms-high-contrast: active) {
+        outline: rem(1px) solid ms-high-contrast-color('text');
+        &::before {
+          display: none;
+        }
+      }
     }
 
     &:focus .Title::before {
       background: var(--p-border-hovered);
     }
+
+    &:active .Title::before {
+      background: var(--p-surface-primary-selected-pressed);
+    }
   }
 
   .DisclosureActivator {
-    margin: 1px 0 -1px;
-    padding: spacing(tight) spacing(tight) spacing(tight) 0;
+    &:hover .Title {
+      background: none;
+
+      &::before {
+        background: var(--p-border-hovered);
+      }
+
+      @media (-ms-high-contrast: active) {
+        outline: rem(1px) solid ms-high-contrast-color('text');
+
+        // stylelint-disable-next-line max-nesting-depth
+        &::before {
+          display: none;
```

Comment:
Rather than set `display: none`, let's actually do `content: none;` and completely remove the pseudo element.